### DISCUSSION TITLE
Should I transpile mobx6 before use it on IE11

### DISCUSSION THREAD

**Discussion by jeiea** [mobxjs/mobx#2853] [Q&A] [answered]
I just tried migration to mobx6, and stuck on `Object.entries` usage of `mobx.esm.js`.
I can try core-js but is it intended usage?

**Comment by danielkcz** [comment, marked as answer]
`Object.entries` is an environmental feature and in the case of IE11 you have to polyfill it. Good service is polyfill.io, but it depends on your needs and requirements. You can surely find sole polyfill.
